src/components/Details.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable react/prop-types */ // TODO: upgrade to latest eslint tooling
```

Comment:
- [ ] I will suggest you fix this eslint error. Instead of disabling it. Eslint wants to ensure you are passing the right data to the component.